src/realm/sync/subscriptions.cpp

```diff
@@ -548,7 +558,7 @@ SubscriptionSet MutableSubscriptionSet::commit()
     const auto flx_version = version();
     m_tr->commit_and_continue_as_read();
 
-    mgr->process_notifications(m_state, flx_version, std::string_view(error_str()));
+    mgr->report_progress();
```

Comment:
Notifications will be sent without this call, but the timing would change and that maybe could be a problem.